dev-games/tiled/tiled-1.11.0.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+inherit toolchain-funcs multiprocessing flag-o-matic python-single-r1 xdg
+
+DESCRIPTION="A general purpose tile map editor"
+HOMEPAGE="https://www.mapeditor.org/"
+SRC_URI="https://github.com/mapeditor/tiled/archive/v${PV}/${P}.tar.gz"
+
+LICENSE="BSD BSD-2 GPL-2+"
+SLOT="0"
```

Comment:
While no package currently in ::gentoo tree depends on tiled, this has SONAMEs specified so we should probably introduce a subslot here for tiled's soname.

Reply:
Tiled libraries don't have sonames specified:
```
> ls /lib64 | grep tiled
libtilededitor.so
libtiled.so
tiled
```